src/components/players/Players/Players.js

```diff
@@ -98,7 +109,92 @@ export default class Players extends Component {
             </table>
           </div>
         </Tab>
+        <Tab label={this.context.t('players.alle')}>
+          <Accordion style={{marginTop: 10}}>
+            <Panel header={this.context.t('players.vttlTeamA')} eventKey="1">
+              {this._getPlayersOfTeam('Vttl','A')}
+            </Panel>
+            <Panel header={this.context.t('players.vttlTeamB')} eventKey="2">
+              {this._getPlayersOfTeam('Vttl','B')}
+            </Panel>
+            <Panel header={this.context.t('players.vttlTeamC')} eventKey="3">
+              {this._getPlayersOfTeam('Vttl','C')}
+            </Panel>
+            <Panel header={this.context.t('players.vttlTeamD')} eventKey="4">
+              {this._getPlayersOfTeam('Vttl','D')}
+            </Panel>
+            <Panel header={this.context.t('players.vttlTeamE')} eventKey="5">
+              {this._getPlayersOfTeam('Vttl','E')}
+            </Panel>
+            <Panel header={this.context.t('players.vttlTeamF')} eventKey="6">
+              {this._getPlayersOfTeam('Vttl','F')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamA')} eventKey="7">
+              {this._getPlayersOfTeam('Sporta','A')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamB')} eventKey="8">
+              {this._getPlayersOfTeam('Sporta','B')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamC')} eventKey="9">
+              {this._getPlayersOfTeam('Sporta','C')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamD')} eventKey="10">
+              {this._getPlayersOfTeam('Sporta','D')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamE')} eventKey="11">
+              {this._getPlayersOfTeam('Sporta','E')}
+            </Panel>
+            <Panel header={this.context.t('players.sportaTeamF')} eventKey="12">
+              {this._getPlayersOfTeam('Sporta','F')}
+            </Panel>
+          </Accordion>
+        </Tab>
       </Tabs>
     );
   }
+
+  _getPlayersOfTeam(competition,team){
+    var team = this.props.teams.filter(x => x.competition === competition && x.teamCode === team).toArray();
+    var spelers = team[0].players;
+    var spelersAsPlayerObject = [];
+    for (var i = 0; i < spelers.length; i++){
+       spelersAsPlayerObject.push(storeUtil.getPlayer(spelers[i].playerId));
+    }
```

Comment:
Je kan ook doen:
var spelersAsPlayerObject = team[0].players.map(player => storeUtil.getPlayer(player.Id));

Probeer ook alles in het engels te houden.
Er is wat nederlands in de backend, maar dat is ook langzaam aan het verdwijnen